frameworks/packet-analysis.rst

```diff
@@ -149,6 +149,21 @@ configuration values from script-land, this can be implemented by overriding
 the ``Initialize()`` method. When overriding this method, always make sure to
 call the base-class version to ensure proper initialization.
 
+With the addition of the transport-layer analyzer to the packet analysis framework,
+it's now possible to register for ports as the identifier. This is natural, given
```

Comment:
I think an quick example use case would help explain this here, not sure it's *that* natural. :)